package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@brainhubeu/react-file-input",
-  "version": "0.7.1",
+  "version": "0.7.2",
```

Comment:
This version was updated by me manually but new renovate PRs will bump automatically and if another PR is merged before merging renovate PR, renovate will rebase its PR.